charts/node-analyzer/templates/daemonset-node-analyzer.yaml

```diff
@@ -40,6 +40,7 @@ spec:
             name: {{ include "agent.configmapName" . }}
             optional: true
         {{- end }}
+        {{- if include "nodeAnalyzer.deployRuntimeScanner" . }}
```

Comment:
It seems some of the volumes defined here (live `vardata-vol` and `varrun-vol`) are used also in other services, such as the `imageAnalyzer`.

Reply:
Nice catch! 👍